### PR TITLE
Fixed issues and added proper Maven support

### PR DESCRIPTION
**Pull request by NULLYUKI**
Following changes were made that will make it possible again to clone and open the project:
- Deleted `.idea` folder from Repo
- Updated `.gitignore` file to prevent issues and removed incorrect and useless entries
- Updated `pom.xml` to properly work when building the JAR file.
- Updated `README.MD` to guide the user on how to build the project.

Please consider merging these changes as soon as possible to fix these issues and add quality of life changes to the project.

### CODE REVIEW

File: .gitignore
Comment (HerXayah) [inline .gitignore:1]: config.json should be put back in.

File: pom.xml
Comment (HerXayah) [inline pom.xml:1]: Dont remove the Exe Compilation.

File: .gitignore
Comment (NULLYUKI) [reply .gitignore:1]: Why? In the readme, it is stated that the config.json is needed.

File: pom.xml
Comment (NULLYUKI) [reply pom.xml:1]: According to the readme, the user has to use the jar file.

**Comment by HerXayah** [comment]
Tldr. Not needed.
Thanks for trying though

File: pom.xml
Comment (HerXayah) [reply pom.xml:1]: > According to the readme, the user has to use the jar file.

According to my maven file, the exe is compiled by building the Jar file.

Comment (NULLYUKI) [reply pom.xml:1]: Its still also in your readme where the user has to use the jar file. And personally it would be much better to provide the program as a JAR file instead of a exe.
